apps/atqr/api/src/app/errors/mailgunErrors.ts

```diff
@@ -0,0 +1,23 @@
+import { Exception } from '@tokilabs/lang';
+
+export default class MailgunError extends Exception {
+  private constructor(
+    message: string,
+    public error: Error,
+    public type: ValidationErrorTypes
+  ) {
+    super(message);
+  }
+
+  static ConfigurationError(error: Error) {
```

Comment:
This is a weird pattern. The method name doesn't make it clear it's a factory. A better approach would be create a `MailgunConfigError` class which extends `MailgunError`